app/actions/actionTypes.js

```diff
@@ -1,2 +1,5 @@
 export const LOAD_USER_SUCCESS = 'LOAD_USER_SUCCESS';
 export const GET_USER_SUCCESS = 'GET_USER_SUCCESS';
+export const CREATE_USER_SUCCESS = 'CREATE_USER_SUCCESS';
+export const UPDATE_USER_SUCCESS = 'UPDATE_USER_SUCCESS';
+export const DELETE_USER_SUCCESS = 'DELETE_USER_SUCCESS';
```

Comment:
@matiasbeckerle These were the constants I was referring to. I made a comment in the file so we can do a threaded discussion hehe.